src/Domain/CoordinateMaps/CubicScale.hpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/TypeTraits.hpp"
+/// \cond
+class FunctionOfTime;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace CoordMapsTimeDependent {
+/*!
+ * \ingroup CoordMapsTimeDependentGroup
+ * \brief CubicScale map defined by \f$x = a(t)*\xi+(b(t)-a(t))\xi^3/X^2\f$,
+ *  where \f$X\f$ is the outer boundary.
+ *
+ * The map scales the coordinates \f$\xi\f$ near the center by a factor
+ * \f$a(t)\f$, while the coordinates near the outer the boundary \f$X\f$,
```

Comment:
remove a `the` from `the outer boundary` - also clarify that X is the coordinate where the map no longer depends on `a` - also comment on conditions for invertibility